soccer/gameplay/root_play.py

```diff
@@ -30,6 +30,11 @@ def __init__(self):
         self._currently_restarting = False
         self._robots: List[robocup.Robot] = []
 
+        ##Counts the number of times a given play has been dropped
+        self.play_drop_count = dict()
+        ##Once a play has been dropped more than this number of times, it won't be considered to be run in the future
+        self.play_drop_threshold = 3
```

Comment:
Do we want this to be a set number for the entire play base. I could also see this being a thing that each play has its own threshold for.

Reply:
Also how long does blacklisting a play in this manner last, like will it never be picked for the rest of the match?

Reply:
So I think we have decided to remove this from this PR and move it to another one to evaluate if we want something more advanced. But this simple implementation just keeps track of all the plays in the play registry and only resets the count when there is a frame where there are no valid plays. If we port this to another PR it would also work that way by default. We could have it decay or something but realistically I think that since our matches aren't that long anyway and we can call time outs to fix issues, if a play is crashing more than a few times we should just kill it and risk a less optimal play running rather than give more chances to a play that is crashing over and over. We should ideally have quite a few plays overall so the loss of a play or two shouldn't really affect us that much.